src/EntityGraphQL/Schema/MutationType.cs

```diff
@@ -176,5 +177,46 @@ public ArgType GetArgumentType(string argName)
             }
             return argumentTypes[argName];
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="entity"></param>
+        private void VaildateModelBinding(object entity)
+        {
+            Type argType = entity.GetType();
+
+            foreach (var prop in argType.GetProperties())
+            {
+                object value = prop.GetValue(entity, null);
+
+                //Did this way so we won't have to reference the DLL
```

Comment:
Is it an issue if we reference DLL?

Reply:
No real concern, just thought you may want to keep the dependencies to a minimum. Whichever you feel is the best.